pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  
+  <groupId>org.dataconservancy.pass</groupId>
+  <artifactId>pass-client</artifactId>
+  <packaging>pom</packaging>
+  <version>0.0.1-SNAPSHOT</version>
+  
+  <name>PASS Client Tool</name>
+  
+  <modules>
+    <module>pass-model</module>
+    <module>pass-json-adapter</module>
+    <module>pass-fedora-client</module>
+    <module>pass-client-api</module>
+    <module>pass-model-ext-nihms</module>
+    <module>pass-client-util</module>
+  </modules>
+  
+  <properties>
+  	<maven-compiler-plugin.version>3.6.1</maven-compiler-plugin.version>
```

Comment:
Please use spaces, not tabs, in all files (there are a number of files that use tabs instead of spaces).

Reply:
hmm... i don't know why eclipse is ignoring my configuration... I noticed it does it inconsistently